spec/order_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'order'
+require 'menu_module'
+
+describe Order do
+
+  subject(:order) { described_class.new }
+
+  let(:confirmation) { double :confirmation }
+
+  full_menu = Menu::STARTERS.merge(Menu::MAINS).merge(Menu::DESSERTS).merge(Menu::SIDES)
+
+  it "has an order class" do
+    expect(order).to be_a Order
+  end
+
+  context "using Menu module" do
+    it "can display starters in a hash" do
+      expect(order.starters).to eq Menu::STARTERS
+    end
+
+    it "can display mains in a hash" do
+      expect(order.mains).to eq Menu::MAINS
+    end
+
+    it "can display desserts in a hash" do
+      expect(order.desserts).to eq Menu::DESSERTS
+    end
+
+    it "can display sides in a hash" do
+      expect(order.sides).to eq Menu::SIDES
+    end
+
+    it "can display a full menu in a hash" do
+      expect(order.view_menu).to eq full_menu
+    end
+  end
+
+  context "when placing orders" do
+    it "has an empty hash for a new order" do
+      expect(order.current_order).to eq []
+    end
+
+    it "can accept an item for an order (starter), specifying quantity and price" do
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)